File: ympa_fsharp_giraffe_server/src/model/EnrichedOrderBoxLayoutDTO.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.OrderBoxLayoutItemDTO

module EnrichedOrderBoxLayoutDTO =

  //#region EnrichedOrderBoxLayoutDTO


  type EnrichedOrderBoxLayoutDTO = {
    Items : OrderBoxLayoutItemDTO[];
    BoxId : int64;
  }
  //#endregion

File: src/main/scala/org/openapitools/server/model/EnrichedOrderBoxLayoutDTO.scala
package org.openapitools.server.model


/**
 * Информация о коробке.
 *
 * @param items Список товаров в коробке.  Если в коробке едет часть большого товара, в списке может быть только один пункт.  for example: ''null''
 * @param boxId Идентификатор коробки. for example: ''null''
*/
final case class EnrichedOrderBoxLayoutDTO (
  items: Seq[OrderBoxLayoutItemDTO],
  boxId: Option[Long] = None
)

